.github/workflows/bazel.yml

```diff
@@ -0,0 +1,21 @@
+name: Bazel CI
+on: [push, pull_request, workflow_dispatch]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    uses: bazel-contrib/.github/.github/workflows/bazel.yaml@v7
+    with:
+      folders: |
+        [
+          ".",
+          "bazel/integration_test",
+        ]
+      exclude: |
+        [
+          {"folder": ".", "bzlmodEnabled": false},
+          {"folder": "bazel/integration_test", "bzlmodEnabled": false},
```

Comment:
I haven't looked at the rules bug `bzlmodEnabled` sounds not like it is testing Bzlmod support. Consider adding a comment explaining why this needs to be like this.

Reply:
These are skipping tests with bzlmod disabled  (that is, only testing with bzlmod), which is invoked here: https://github.com/bazel-contrib/.github/blob/master/.github/workflows/bazel.yaml#L121

At this point, I don't think it makes sense to add support for workspaces.  I will add a note.

Reply:
I might have a logical twist somewhere but I read the linked line as: If `bzlmodEnabled==true` then `--enable_bzlmod=true --enable_workspace=false`. So false is the opposite which we don't seem to desire.

Reply:
> I might have a logical twist somewhere 

This is in the "exclude" section.  So we are excluding the test configurations where bzlmodEnabled=false.  I agree that it's confusing.

Reply:
Thanks for the clarification!